commons-core/src/main/scala/com/avsystem/commons/SharedExtensions.scala

```diff
@@ -176,6 +180,9 @@ object SharedExtensions extends SharedExtensions {
 
     def ensurePrefix(prefix: String): String =
       if (str.startsWith(prefix)) str else prefix + str
+
+    def indent(ind: String): String =
```

Comment:
Docs, unit tests?

Reply:
I'll move it to tests just to avoid writing docs and tests :trollface: